specification/dartLangSpec.tex

```diff
@@ -15671,7 +16105,7 @@ \subsection{Type Void}
   }
 \item
   \commentary{%
-  In an arrow function body \code{=> $e$},
+  In an arrow function body `\code{=> $e$}',
```

Comment:
Reason for change: For readability, we use single quotes around some source code snippets that may otherwise be hard to read. Here: `=>` and `e` separated by a space is not obviously a thing (it looks like two things).